lib/chromedriver.js

```diff
@@ -216,22 +216,33 @@ class Chromedriver extends events.EventEmitter {
 
   async killAll () {
     let cmd;
+    let cmdAdb;
     if (system.isWindows()) {
       // js hint cannot handle backticks, even escaped, within template literals
       cmd = "FOR /F \"usebackq tokens=5\" %a in (`netstat -nao ^| " +
             "findstr /R /C:\"" + this.proxyPort + " \"`) do (" +
             "FOR /F \"usebackq\" %b in (`TASKLIST /FI \"PID eq %a\" ^| " +
             "findstr /I chromedriver.exe`) do (IF NOT %b==\"\" TASKKILL " +
             "/F /PID %a))";
+      cmdAdb = "FOR /F \"usebackq tokens=2\" %a in (`adb forward --list ^| findstr /R /C:\"webview\"`) do ( adb forward --remove %a )";
     } else {
       cmd = `pkill -15 -f "${this.chromedriver}.*--port=${this.proxyPort}"`;
+      cmdAdb = "for a in `adb forward --list | grep \"webview\" | awk '{print $2}'` ; do  adb forward --remove $a ; done";
     }
-    log.info(`Killing any old chromedrivers, running: ${cmd}`);
+    const cp_exec = B.promisify(cp.exec);
+    log.debug(`Killing any old chromedrivers, running: ${cmd}`);
     try {
-      await (B.promisify(cp.exec))(cmd);
-      log.info("Successfully cleaned up old chromedrivers");
+      await cp_exec(cmd);
```

Comment:
The adbPort past in constructor parameters is not the adb forward port that ADB listened on. It seems that chromedriver will ask ADB to use an other random port. 
I use appium-adb to find ADB binary path.